rfcs/20220523-depthwise-post-op-extension/README.md

```diff
@@ -0,0 +1,63 @@
+# RFC: Depthwise post-op extension
+
+## Introduction
+oneDNN provides a performance optimization for a chain of convolutions, when 1x1 convolution is followed by 3x3 depthwise convolution with stride of 1 or 2.
+This optimization is provided through an attribute post-op API.
+To keep usability in place, the decision in the past was to hardcode depthwise convolution settings into a post-op name.
+Thus, `k3s1p1` or `k3s2p1` options are only allowed.
+This optimization targeted only MobileNet topologies popular back then.
+Now there are models that have same pattern of 1x1 + depthwise, but they might have different depthwise convolution settings.
```

Comment:
semi random spot: I assume we will be deprecating the existing dw-post API in the next major version of oneDNN?

Reply:
Well, that would make a lot of sense. Let's add this to the final text as well, thanks for bringing this up.